pkg/sentry/kernel/task_signals_arm64.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build arm64
+
+package kernel
+
+import (
+	"gvisor.dev/gvisor/pkg/sentry/arch"
+)
+
+// updateArchAct updates the arch-specific features.
+// The vdso is enabled, we can use the sigreturn trampolines
+// the vdso provides instead.
+// Please see the linux code as reference:
+// linux/arch/arm64/kernel/signal.c:setup_return()
```

Comment:
we can use the vdso trampoline only  SA_RESTORER isn't set. Otherwise we have to use sa_restorer which is set by a user application. In some cases, sa_restorer can be implemented in user-space without calling the rt_sigreturn syscall.

Reply:
Hi @avagin 
SA_RESTORER = 0x4000000 will be only used on Intel platform. This flag is unused on other platforms. 
On other platform, it will be defined as this:
SA_RESTORER = 0x0

Reply:
https://github.com/torvalds/linux/blob/master/arch/arm64/kernel/signal.c#L735

Reply:
Done.

Reply:
Pls update the comment. "SA_RESTORER is unused on ARM64" is wrong. I am involved in the CRIU project where we use SA_RESTORER on arm64, arm, ppc64le, x86_64 and s390 and it works on all these platforms.

Reply:
Done.